client/src/components/pages/MainFriend.tsx

```diff
@@ -95,17 +106,27 @@ const FriendsFragment: FC<FriendsFragmentProps> = ({friendsKey}) => {
     );
   }, [data]);
 
-  const userListOnPress = (user: ProfileModal_user$key): void => {
+  const userListOnPress = (
+    user: ProfileModal_user$key,
+    isMyself: boolean,
+  ): void => {
     showModal({
       user,
+      isMyself,
     });
   };
 
   const renderItem: ListRenderItem<typeof nodes[number]> = ({item}) => {
-    const userListOnPressInlineFn = (): void => userListOnPress(item);
+    const isMyself = item.id === authUser?.id;
+    const userListOnPressInlineFn = (): void => userListOnPress(item, isMyself);
 
     return (
-      <UserListItem showStatus user={item} onPress={userListOnPressInlineFn} />
+      <UserListItem
+        showStatus
+        user={item}
+        onPress={userListOnPressInlineFn}
```

Comment:
I think it is okay to inline this function.
```suggestion
        onPress={() => userListOnPress(item, isMyself)}
```